app/views/users/show.html.erb

```diff
@@ -1,11 +1,36 @@
-<h1><%= @user.name %></h1>
-
-
+<div class="container">
+<div><div class="aside">
+    <div class="aside__left">
+    <div class="aside__left__responsive">
+    <img src= <%=  @user.photo %> alt= <%=  @user.id %> class="responsive aside__left__responsive__respo">
+    </div>
+    </div>
+    <div class="aside__right">
+    <%=  @user.name %>
+    <%=  @user.postsCounter %>
+    </div>
+    </div></div>
+<div class="bio">
+<span>Bio</span><br>
+ <%=  @user.bio %>
+</div>
+</div>
 
 <h2>Posts:</h2>
 
-<ul>
+<ul class="useridposts">
   <% @user.posts.each do |post| %>
-    <li><%= link_to post.title, user_post_path(@user, post) %></li>
+   <div class="container">
+    <li>
```

Comment:
- [ ] Here we do not need all the posts, fetching all the posts should be done using the button `see all posts`. Kindly make use of the `recent_posts` callback function you created in the user model to render the most recent 3 posts.